src/drivers/pmw3901/pmw3901.cpp

```diff
@@ -575,7 +575,7 @@ PMW3901::collect()
 	_flow_sum_x += delta_x_raw;
 	_flow_sum_y += delta_y_raw;
 
-	if (_flow_dt_sum_usec < 45000) {
+	if (_flow_dt_sum_usec < 15000) {
```

Comment:
What is this? Please document magic numbers.

Reply:
the collect cycle waits 15000 usecs to collect sensor data and continues after it. the number determines the publishing rate. 
it is now updated as _collect_time